env-template

```diff
@@ -1,2 +0,0 @@
-# TODO: Scope out additional ENV necessary for dev/prod/test and create script to generate safe defaults
```

Comment:
Don't delete this file. I created it to represent what your local `.env` should look like. 

FYI - our actual `.env` files is what should be ignored in `.gitignore` and never committed; these files typically contain sensitive information that is essential for the application to run (e.g., API secrets, DB password, etc.).

Reply:
Reinstated the file. Should be good to go.